challenge.md

```diff
@@ -6,37 +6,37 @@ Write your answers inside this file, where it's indicated by the comments.
 
 1. We just forked a repo on GitHub.com and want to start working on it locally. What command do we use to do that?
 
-<!-- Answer Starts Here -->
+<!-- After you have forked the repo, you have to copy the SSH clone URL. Then, in the command line, you must go to the directory where you would like this new repo. There, you will make sure you're not in another git repo. Once you're safely not in another git repository, you would type 'git clone + the SSH clone URL' to give yourself a local repo to work on. -->
 
 <!-- Answer Ends Here -->
 
 2. OK, we just wrote some code. What command could we use to see a list of all the changes that have been made since the last commit?
 
-<!-- Answer Starts Here -->
+<!-- You could use the command 'git status' to see changes that have been made since the last commit. This would show both staged and unstaged changes.  -->
```

Comment:
True. However, it wouldn't show the specific changes to any given file. For that, you'd also want to use `git diff` to look at the content of your changes. While it's true that `git diff` will only show you unstaged changes, you should be looking at the details of the changes you plan to make to a file _before_ you `add` that file.